### PR TITLE
Adding LDAP authentification option

### PR DESCRIPTION
**Pull request by marianschmotzer**
Adding LDAP auth lines to configuration file. This will enable LDAP as source of users/groups when ldap plugin is installed.

### CODE REVIEW

**Comment by lrk** [comment]
I should also fix the automated tests before merging your PR.

**Comment by lrk** [comment]
I fix the test, you can rebase your branch on master.

**Comment by marianschmotzer** [comment]
Hi
i will fix mentioned things.

pi 13. 9. 2019 o 11:16 Cengis K. <notifications@github.com> napísal(a):

File: templates/sonar.properties.j2 [hunk 1]
```diff
@@ -40,6 +40,46 @@ sonar.web.http.minThreads={{ sonar_web_http_min_threads | default('5') }}
 sonar.web.http.acceptCount={{ sonar_web_http_accept_count | default('25') }}
 sonar.ajp.port={{ sonar_ajp_port | default('-1') }}
 
+#--------------------------------------------------------------------------------------------------
+# LDAP AUTH
+{% if sonar_ldap is defined %} 
+sonar.security.realm=LDAP
+{% if sonar_ldap.authenticator_downcase is defined %} 
+sonar.authenticator.downcase={{ sonar_ldap.authenticator_downcase }}
```
[the remark this answers is not in the document]
Comment (marianschmotzer) [reply templates/sonar.properties.j2]: Requested changes are in new commit.

**Pull request merged**